credit_card.html

```diff
@@ -0,0 +1,53 @@
+<fieldset class="input-group cc-fields">
+    <div id="ak-fieldbox-card_num" class="input-text ak-input-type-user cc-fields">
+      <label for="ak-card_num">{% filter ak_text:"field_card_num" %}Credit Card
+        #{% endfilter %}</label>
+      <input type="hidden" name="required" value="card_num" id="ak-card_num-required">
+      <div id="ak-card_num-hosted" class="hosted-field"></div>
+      <input id="ak-card_num" type="text" name="card_num">
+    </div>
+    <div class="inline_fields">
+      <div id="ak-fieldbox-exp_date"
+        class="input-select input-cc-expiration ak-input-type-user cc-fields ak-err-below">
+        <input type="hidden" name="required" value="exp_date_month" id="ak-exp_date_month-required">
+        <label for="ak-exp_date_month">{% filter ak_text:"field_exp_date" %}Expiration
+          Date{% endfilter %}</label>
+        <div id="ak-exp_date-hosted" class="hosted-field"></div>
+        <select id="ak-exp_date_month" class="input-cc-expiration-month c4 ct4 cm4 no-margin"
+          type="text" name="exp_date_month">
+          <option value="">MM</option>
+          <option value="01">01</option>
+          <option value="02">02</option>
+          <option value="03">03</option>
```

Comment:
super nit: we could rewrite this as a loop to avoid the repetition. Feel free to ignore this comment though with credance to time